src/error.rs

```diff
@@ -71,6 +71,7 @@ pub(super) enum Parse {
     #[cfg(feature = "http1")]
     VersionH2,
     Uri,
+    UriTooLong,
```

Comment:
Perhaps the `Error::is_parse_too_large()` should return true for this variant as well?

Reply:
Thanks for the feedback. Pull request has been updated with the changes.

Reply:
Sorry, forgot to add the file. Pull request has now been updated.